vmm/src/lib.rs

```diff
@@ -786,6 +786,40 @@ pub struct VmmThreadHandle {
     pub http_api_handle: Option<HttpApiHandle>,
 }
 
+/// Describes the current ownership of the VM.
+#[allow(clippy::large_enum_variant)]
+pub enum MaybeVmOwnership {
+    /// The VMM holds the ownership of the VM.
+    Vmm(Vm),
+    /// The VM is temporarily by the current ongoing migration.
```

Comment:
Can't you give suggestions like so?
```suggestion
/// The VM is temporarily blocked by the current ongoing migration.
```